episodes/lesson-content.md

```diff
@@ -72,7 +72,7 @@ from how it appears on your website, or not at all.
 
 We will encounter more examples of this throughout the lesson,
 but the important thing to remember for now is that
-**how GitHub displays source (R) Markdown files is not a reliable indicator
+**how GitHub displays source (R)Markdown files is not a reliable indicator
```

Comment:
```suggestion
**how GitHub displays source (R) Markdown files is not a reliable indicator
```

I think the space between the R and the M is [how we are supposed to write it](https://rmarkdown.rstudio.com/).

Reply:
Ah, I think we may then need to change this elsewhere for consistency. Thanks @tobyhodges.